definitions/npm/lambda-log_v2.x.x/flow_v0.112.x-/lambda-log_v2.x.x.js

```diff
@@ -0,0 +1,95 @@
+import { WriteStream } from "fs";
+import { Console } from "console";
+import { EventEmitter } from "events";
+
+declare module "lambda-log" {
+  declare interface LogRecordOptions {
+    level: string;
+    msg: string;
+    meta?: any;
+    tags?: string[];
+  }
+  declare interface LogRecord {
+    msg: string;
+    meta: any;
+    _logLevel: string;
+    _tags: string[];
+  }
+  declare export class LogMessage {
+    level: string;
+    msg: string;
+    meta?: any;
+    tags?: string[];
+    constructor(logRecordOptions: LogRecordOptions, opts: LambdaLogOptions): this;
+    value: LogRecord;
+    log: LogRecord;
+    throw: void;
+    toJSON(format?: boolean): string;
+    static isError(val: any): boolean;
+  }
+  declare interface LambdaLogOptions {
+    meta?: any;
+    tags?: string[];
+    dynamicMeta?: (message: LogMessage) => any;
+    debug?: boolean;
+    dev?: boolean;
+    silent?: boolean;
+    replacer?: (key: string, value: any) => any;
+    stdoutStream?: WriteStream;
+    stderrStream?: WriteStream;
+  }
+
+  declare type ConsoleMethods = $Keys<typeof console>;
+
+  // The Log Levels object defines custom levels. The value of each level is either a console method name or a function that returns a console method name. https://www.npmjs.com/package/lambda-log#log-levels-object
+  declare export type LogLevelsParam = {
+    [string]: ConsoleMethods | (message) => ConsoleMethods | boolean,
+    ...
+  };
+
+  declare export type DefaultLogLevels = {|
+    info: 'info',
+    warn: 'warn',
+    error: 'error',
+    debug: false
+  |};
+
+  declare type GetLevelMethod = <V>(V) => ((msg: string, meta?: { ... }, tags?: string[]) => LogMessage);
+  declare type CustomLogLevelMethods<LogLevels: LogLevelsParam> = $ObjMap<LogLevels, GetLevelMethod>;
+
+  declare export type LambdaLog<LogLevels: LogLevelsParam> = {|
+
+    // extends EventEmitter
+    addListener(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    on(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    once(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    prependListener(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    prependOnceListener(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    removeListener(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    off(event: string | symbol, listener: (...args: any[]) => void): LambdaLog<LogLevels>,
+    removeAllListeners(event?: string | symbol): LambdaLog<LogLevels>,
+    setMaxListeners(n: number): LambdaLog<LogLevels>,
+    getMaxListeners(): number,
+    listeners(event: string | symbol): Function[],
```

Comment:
`Function` is deprecated, and is equal to `any`. Would be nice if you could type it a bit more precisely.

Reply:
Thanks. I replaced the deprecated `Function` keyword and tightened the related types.